source/clear-linux/guides/maintenance/enable-user-space.rst

```diff
@@ -0,0 +1,177 @@
+.. _enable-user-space:
+
+Create and enable a new user space
+##################################
+
+This section provides steps to complete the following basic setup tasks for
+a newly installed |CL| system:
+
+* Create a new user
+* Update the OS to its most current version using `swupd`.
+* Install the most common applications for system administrators and
+  developers using bundles.
+* Setup a new user.
+* Setup `sudo` privileges for that new user.
+* Install a GUI using those `sudo` privileges.
+
+.. note::
+   You must be logged in as the root user to complete the tasks in this
+   section.
+
+Create a new user
+=================
+
+To create a new user and set a password for that user, enter the following
+command as a root user:
+
+.. code-block:: console
+
+   useradd <userid>
+   passwd <userid>
```

Comment:
The password should note be the same as the userid.  I would use <password>

Reply:
Not sure what you want here.  We are only showing the commands needed to create a new userid and then set a password for that new userid

Reply:
The value for passwd should not be <userid> it makes the example confusing.

Reply:
I meant that the value for `passwd` should not be <userid>. It makes the example confusing.

Reply:
Hopefully this is more clear

Reply:
@tpleavitt Please replace <userid> on line 30 with <password>, or something similar:
.. code-block:: console
 
   useradd <userid>
   passwd <password>

Reply:
Since the user is root, they need to specify the user that they will add as well as the passwd for that user.

Reply:
The content is already merged and all the mentioned issues addressed.